apps/docs/content/components/concepts/internationalization.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Internationalization
+description: This page describes how adapting components to respect languages and cultures of users around the world is an important way to help make your application accessible to the widest number of people.
+order: 8
+---
+
+## Introduction
+
+Internationalization is the process of structuring your code and user interface to support localization. Hopper supports many aspects of localization for many components out of the box, including translations for builtin strings, localized date and number formatting, and more. By using Hopper to build your applications, these aspects of internationalization are handled for you.
+
+To set the locale for your application, you can follow the procedure described in the [HopperProvider documentation](../application/hopper-provider#locales).
+
+## Supported locales
+
+Hopper currently supports translations for 2 locales. They are listed below.
+
+- English (United States) - `en-US`
+- French (Canada) - `fr-CA`
+
+## Optimizing bundle size
+
+Hopper uses React-aria for internationalization, which is a peer dependency. This means that you must install React-aria in your application. React-Aria includes translations for over 30 languages.
+This is inclusive to the most users out of the box, but comes at the cost of bundle size. Since applications using Hopper does not support all of these locales, you can use React-Aria build plugins to include only the languages that we support in your JavaScript bundle.
```

Comment:
```suggestion
This is inclusive to the most users out of the box, but comes at the cost of bundle size. Since applications using Hopper do not support all of these locales, you can use React-Aria build plugins to include only the languages that we support in your JavaScript bundle.
```